### ISSUE TITLE
Repcred testing - Healthy Controls

### ISSUE THREAD

**Issue by bcorrie** [airr-community/rep-cred#33]
Did the following search on the Gateway:

- Study Group = "Control (Healthy)"
- Sorted from small to large sequence count (so testing was faster since started with the smallest repertoires)
- Manually, for each study from small to large, run repcred on the all of the repertoires from the subject with the smallest sequence count - that is just one subject from each study, and then run repcred on all of the repertoires from that subject. Some subjects of course only have one repertoires, some have several.

Essentially did this search as a somewhat random query to narrow down the repertoires that I test while at the same time getting a decent sample across studies (and therefore annotation tools) and IG/TR loci. Started from small to big so the testing was faster.

Here are the results:

- Study ID: PRJCA002413, Study group: Control (Healthy), Sample ID: Healthy_Control_1 - **IGH, TRB `success`**, **IGK, IGL, TRA `failed`**
  - This is essentially the same problem as #29 
- Study ID: SRP001460, Study group: Control (Healthy), Subject ID: 42 - **Two IGH repertoires `success`**
- Study ID: PRJEB1289, Study group: Control (Healthy), Subject ID: Healthy_4_5-sc-2013 - **IGH `success`**
- Study ID: PRJNA275625, Study group: Control (Healthy), Subject ID: SAMN03352119 -**IGH `success`**
- Study ID: PRJNA280743, Study group: Control (Healthy), Subject ID: PEM1-10 - **IGH `success`**
- Study ID: PRJNA630455, Study group: Control (Healthy), Subject ID: H8, Sample ID: H8-rep2 - **IGH `failed`**
  - This is essentially the same problem as #27 (vj_in_frame missing causing crash)
- Study ID: PRJNA633317, Study group: Control (Healthy), Subject ID: W, Sample ID: W_2018, PCR target: TRA - **TRA `failed`**
  - This is a new problem, see new issue #32

So ran on 12 repertoires, and only found one new problem...

**Comment by ssnn-airr** [comment]
@bcorrie Please try again once the container is ready

**Comment by bcorrie** [comment]
#32 issue has been fixed so #32 and this issue can probably be closed...
